Dockerfile.dapper

```diff
@@ -28,9 +28,14 @@ RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
 RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
         GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0; \
     fi
+## install ginkgo
+RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
+        GO111MODULE=on go get github.com/onsi/ginkgo/ginkgo@v1.14.0; \
+    fi
 # -- for make rules
 
 # -- for dapper
+ENV DAPPER_RUN_ARGS --privileged --network host
```

Comment:
Why do we need the host network? Is it dependent on the network setup? I don't see any prerequisite mentioned though

Reply:
in the dapper container，harvester progress can be able to access the kind cluster  by host network.

Reply:
![image](https://user-images.githubusercontent.com/15064560/100042194-eb494200-2e45-11eb-9bdf-f964536eb798.png)